config/runtimes/kserve-huggingfaceserver-multinode.yaml

```diff
@@ -16,35 +16,33 @@ spec:
     - v1
   containers:
     - name: kserve-container
-      image: kserve/huggingfaceserver:latest
+      image: kserve/huggingfaceserver:latest-gpu
       command: ["bash", "-c"]
       args:
         - |
-          ray start --head --disable-usage-stats --include-dashboard false 
-          # wait for other node to join
-          until [[ $(ray status | grep -c node_) -eq ${PIPELINE_PARALLEL_SIZE} ]]; do
-            echo "Waiting..."
-            sleep 1
-          done
-          ray status
-
           export MODEL=${MODEL_ID}
           if [[ ! -z ${MODEL_DIR} ]]
           then
-            MODEL=${MODEL_DIR}
+            export MODEL=${MODEL_DIR}
           fi
 
-          python3 -m huggingfaceserver --model_name=${MODEL_NAME}  --model_dir=${MODEL} --tensor-parallel-size=${TENSOR_PARALLEL_SIZE} --pipeline-parallel-size=${PIPELINE_PARALLEL_SIZE}
+          export RAY_ADDRESS=${POD_IP}:${RAY_PORT}
+          ray start --head --disable-usage-stats --include-dashboard false 
+          python ./huggingfaceserver/health_check.py registered_nodes --retries 200  --probe_name runtime_start
```

Comment:
Why it needs 200 retries? some comments might be helpful

Reply:
Previously, there is no timeout so I added 200s by adding 200 retries. 
Basically, I don't think it would take more than 10~30 seconds but I would give enough time to wait.